packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-fulfilment/__tests__/route.spec.js

```diff
@@ -41,5 +41,17 @@ describe('licence-fulfilment > route', () => {
       const result = await getData(getMockRequest())
       expect(result.youOrOther).toEqual(returnedValue)
     })
+
+    it.each([
+      ['true', true],
+      ['false', false],
+      [undefined, false]
+    ])('SHOW_NOTIFICATION_BANNER is set to value of process.env.SHOW_NOTIFICATION_BANNER', async (notification, expectedResult) => {
```

Comment:
The test name will be the same for all three iterations, it should differ so that we know what's being tested. "when SHOW_NOTIFICATION_BANNER env var is %s, flag for showing notification banner should be %s".